packages/client/public/index.html

```diff
@@ -2,19 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <meta
-      http-equiv="Content-Security-Policy"
-      content="
-        default-src 'self';
-        connect-src 'self' ws://localhost:3000;
-        script-src 'self' chrome-extension:;
-        style-src 'self' 'unsafe-inline' http://fonts.googleapis.com;
-        img-src 'self' data: mid:;
-        child-src 'self' data:;
-        font-src 'self' http://fonts.gstatic.com;
-        media-src 'self';
-        form-action 'none';
-        " />
+    <meta http-equiv="Content-Security-Policy" content="" />
```

Comment:
The CSP is now generated by Webpack, not written by hand. We put in a `meta` tag with an empty `content` value to show Webpack where to emit the policy.